lib/python-beta/src/bailo/helper/release.py

```diff
@@ -0,0 +1,214 @@
+    :param files: (optional) A list of files for release
+    :param images: (optional) A list of images for release
+    :param minor: Is a minor release?
+    :param draft: Is a draft release?
+
+    ..note:: Currently files and images are stored as string references
+    """
+    def __init__(
+        self,
+        client: Client,
+        model_id: str,
+        version: Version | str,
+        model_card_version: float = 1,
+        notes: str = "",
+        files: list[str] = [],
+        images: list[str] = [],
+        minor: bool = False,
+        draft: bool = True,
+    ) -> None:
+
+        self.client = client
+        self.model_id = model_id
+
+        if type(version) == str:
+            version = Version(version)
+        self.version = version
+
+        self.model_card_version = model_card_version
+        self.minor = minor
+        self.notes = notes
+        self.files = files
+        self.images = images
+        self.draft = draft
+        self.files = files
+
+    @classmethod
+    def create(
+        cls,
+        client: Client,
+        model_id: str,
+        version: Version | str,
+        model_card_version: float,
+        notes: str = "",
+        files: list[str] = [],
+        images: list[str] = [],
+        minor: bool = False,
+        draft: bool = True,
+    ) -> Release:
+        """ Builds a release from Bailo and uploads it
+
+        :param client: A client object used to interact with Bailo
+        :param model_id: A Unique Model ID
+        :param version: A semantic version of a model release
+        """
+        if type(version) == Version:
+            version = Version(version)
+
+        client.post_release(model_id,model_card_version, str(version), notes, files, images, minor, draft)
+
+        return cls(
+            client,
+            model_id,
+            version,
+            model_card_version,
+            notes,
+            files,
+            images,
+            minor,
+            draft
+        )
+
+
+    @classmethod
+    def from_version(
+        cls,
+        client: Client,
+        model_id: str,
+        version: Version | str
+    ) -> Release:
+        """ Returns an existing release from Bailo
+
+        :param client: A client object used to interact with Bailo
+        :param model_id: A Unique Model ID
+        :param version: A semantic version of a model release
+        """
+
+        res = client.get_release(model_id, str(version))['release']
+
+        model_card_version = res['modelCardVersion']
+        notes = res['notes']
+        files = res['fileIds']
+        images = res['images']
+        minor = res['minor']
+        draft = res['draft']
+
+        return cls(
+            client,
+            model_id,
+            version,
+            model_card_version,
+            notes,
+            files,
+            images,
+            minor,
+            draft
+        )
+
+    def download(self, local_dir: str) -> None:
```

Comment:
Maybe split into two functions, one to download one file and one to download multiple.